### PR TITLE
Revert etcd client version

### PR DESCRIPTION
**Pull request by style95**
## Description
This is to fix https://github.com/apache/openwhisk/issues/5317 by reverting https://github.com/apache/openwhisk/commit/40077664ab3011e7a333b0197fa9a285502a1d81.
It seems the version of gprc-core module in the ETCD client conflicts with akka-grpc module.

I am curious how it passed all checks in the first place, but anyway it would be the easiest way to resolve it for now.

I think we need to upgrade both modules at once.

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Scheduler
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [x] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [x] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md#coding-standards) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Comment by style95** [comment]
ok, I found there are only unit tests in the scheduler CI tests.
This will guarantee the sanity of the critical path of the scheduler.
https://github.com/apache/openwhisk/pull/5308

**Review by ningyougang — APPROVED** [review]

File: common/scala/build.gradle [hunk 1]
```diff
@@ -75,7 +75,7 @@ dependencies {
     api "io.kamon:kamon-datadog_${gradle.scala.depVersion}:2.1.12"
 
     // for etcd
-    api "com.ibm.etcd:etcd-java:0.0.21"
+    api "com.ibm.etcd:etcd-java:0.0.13"
```
Comment (ningyougang) [inline common/scala/build.gradle:78]: Seems we can't use higher version forever?

Comment (style95) [reply common/scala/build.gradle:78]: As I said above, we can upgrade both etcd client and akka-grpc module.

**Comment by style95** [comment]
I merged it as no activation can be invoked with the current master.

**Pull request merged**